### PR TITLE
create static pages

### PR DESCRIPTION
**Pull request by phamhuycuong**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,14 @@
 Rails.application.routes.draw do
+  get 'static_pages/home'
+
+  get 'static_pages/word_list'
```
Comment (phamkykhoi) [inline config/routes.rb]: Remove

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+
+  get 'static_pages/categories'
```
Comment (phamkykhoi) [inline config/routes.rb]: Remove

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+
+  get 'static_pages/lesson'
```
Comment (phamkykhoi) [inline config/routes.rb]: Remove

File: app/controllers/static_pages_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
+
+  def word_list
```
Comment (phamkykhoi) [inline app/controllers/static_pages_controller.rb]: Remove

File: app/assets/javascripts/static_pages.coffee [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/static_pages.coffee]: remove this file. Pls use `git rm` instead remove by your hand

File: app/assets/stylesheets/static_pages.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/static_pages.scss]: remove this file. Pls use `git rm` instead remove by your hand

File: app/helpers/static_pages_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove this file. Pls use `git rm` instead remove by your hand

File: app/controllers/static_pages_controller.rb (hunk 4 above)
Comment (dieunb) [inline app/controllers/static_pages_controller.rb]: I don't understand this function? And i think no need it.

File: app/controllers/static_pages_controller.rb [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def categories
```
Comment (dieunb) [inline app/controllers/static_pages_controller.rb]: I don't understand this function? And i think no need it.

File: app/controllers/static_pages_controller.rb [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def lesson
```
Comment (dieunb) [inline app/controllers/static_pages_controller.rb]: I don't understand this function? And i think no need it.

File: app/controllers/static_pages_controller.rb [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def result
```
Comment (dieunb) [inline app/controllers/static_pages_controller.rb]: I don't understand this function? And i think no need it.

Comment (phamkykhoi) [inline app/controllers/static_pages_controller.rb]: Remove action

File: app/controllers/static_pages_controller.rb (hunk 9 above)
Comment (phamkykhoi) [inline app/controllers/static_pages_controller.rb]: Remove action

File: app/controllers/static_pages_controller.rb (hunk 8 above)
Comment (phamkykhoi) [inline app/controllers/static_pages_controller.rb]: Remove action

File: config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,14 @@
...
+
+  get 'static_pages/result'
```
Comment (phamkykhoi) [inline config/routes.rb]: Remove

**Comment by phamkykhoi** [comment]
2 Comments

File: app/controllers/static_pages_controller.rb [hunk 12]
```diff
@@ -0,0 +1,3 @@
+class StaticPagesController < ApplicationController
+
```
Comment (phamkykhoi) [inline app/controllers/static_pages_controller.rb]: This controller have not method

**Pull request merged**